src/main/java/com/courtcanva/ccfranchise/constants/OrderStatus.java

```diff
@@ -1,9 +1,19 @@
 package com.courtcanva.ccfranchise.constants;
 
 public enum OrderStatus {
-    UNASSIGNED,
-    ASSIGNED,
-    ACCEPTED,
-    COMPLETED,
-    CANCELED
-}
\ No newline at end of file
+    UNASSIGNED("unassigned"),
```

Comment:
it is better to use the code from the main branch to keep the code consistent with other teammates

Reply:
I refined `OrderStatus `to use value enum's value easily